### ISSUE TITLE
Docs refer to Message textbox that no longer exists

### ISSUE THREAD

**Issue by DTronD** [MicrosoftDocs/power-automate-docs#286]
The doc refers to pasting the provided/modified JSON for an Adaptive Card into the Message textbox, but in the current version of the "Post an Adaptive Card to a Teams channel and wait for a response" action, there is no a button for "Create an Adaptive Card"

After clicking on that button, when you return to the Action there is a Message section that has a double textbox row with "Enter key" in the first textbox and "Enter value" in the second textbox.

My attempts to past my own adaptive card JSON into the Adaptive Card Editor tool fail to update the card used.  Do I have to recreate the A.C. using the tool even though I have a working A.C. JSON ready to paste in.

1) Please update the docs to reflect the current form of the action.
2) Please provide details on how to get an existing JSON string into the action or the Adaptive Card Editor tool.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf788f3b-9ad6-3c79-689d-5f36c1ab292f
* Version Independent ID: d8a78d1d-7f90-7fb1-b0f6-8ae6518031f1
* Content: [Create flows that post adaptive cards to Microsoft Teams - Power Automate](https://docs.microsoft.com/en-us/power-automate/create-adaptive-cards)
* Content Source: [articles/create-adaptive-cards.md](https://github.com/MicrosoftDocs/power-automate-docs/blob/live/articles/create-adaptive-cards.md)
* Service: **flow**
* GitHub Login: @MSFTMan
* Microsoft Alias: **deonhe**

**Comment by Dean-Haas** [comment]
Hi DtronD. Thanks for submitting this. I have been trying to reproduce the issue you describe, but I am confused by your reference to a "Create an Adaptive Card" button. I don't see this referenced in the topic, and the UI in power automate matches what I am seeing in the UI. The "post an adaptive card to a teams channel..." action screen contains a message field where the JSON code is pasted. Please let me know if I am misreading your issue here. Thanks!

**Comment by DTronD** [comment]
@Dean-Haas , sorry for the poor explanation. E.g. I should have included a screenshot.  Let me try again....

Ignoring the typos/misstatements in my original, let me start over to better specify the issues and resolution I'm suggesting:

**Issue 1**) The docs page says for step 10 "Paste this JSON into the Message box:"  but I see two text boxes in the Message section. I.e. one with hint text of "Enter key" and another of "Enter value." 
![image](https://user-images.githubusercontent.com/49466287/90651333-7e992880-e202-11ea-8808-19236c90d35d.png)

![image](https://user-images.githubusercontent.com/49466287/90651235-66290e00-e202-11ea-9784-736cde381295.png)

**Issue 1 Request**: Please clarify the docs to indicate where the JSON should be pasted (i.e. Key or Value).

**Issue 2**) Now that I'm seeing the action's "Create Adaptive Card" button at the very moment the action is added to the flow, I still can't seem to get the JSON generated in the Adaptive Card dialog to update based on changes I make there.
![image](https://user-images.githubusercontent.com/49466287/90652628-e2702100-e203-11ea-94f8-e7e971be73b6.png)

I can paste the docs page's example JSON into the dialog's "Card Payload Editor" but that doesn't make the displayed/resultant card update.  It seems the interaction is oneway only (i.e. designer changes code-behind JSON but changing the code-behind JSON does NOT alter the designer).  Furthermore, hitting the "Save Card" button in the Adaptive Card (designer) dialog just returns you to your Flow with no noticeable changes to the Flow action.  If you re-enter the Adaptive Card dialog using the "Edit Adaptive Card" button, any changes you made in the designer (but NOT in the JSON in the "Card Payload Editor") are available....so it IS saving the user-designed card somewhere.
![image](https://user-images.githubusercontent.com/49466287/90657260-367d0480-e208-11ea-9398-2ac1230e3d45.png)


**Issue 2 Request**: The docs page has pointers in the Troubleshooting section regarding using VSCode and the A.C. plug-in.  This is probably a better option than using the Adaptive Card dialog designer.  Assuming that is the case, please clarify the docs for the ONE recommended way of
    (a) designing a custom A.C. for this action type
    (b) pasting in JSON for an A.C. already created elsewhere (e.g. a company-standardized A.C. from source control, the example JSON in the docs, a community-shared AC design).

Also, I realize the Adaptive Card dialog is probably external to the PAutomate and PAutomate docs teams.  But, since the Action has a button that opens it, a pointer in the PAutomate docs to the documentation for that tool would be good.  And, if possible, some warnings about the tool's seeming weirdness on oneway direction of changes in the visual designer --> codeBehindJSON might prevent frustration.

In short, I'm finding it confusing on how to use this action and get my A.C. JSON in use without hand-designing the A.C. every time.  Thanks!

**Comment by Dean-Haas** [comment]
@hamenon I think we looked at this. Can you let me know if a docs update is needed?  Thanks.

**Comment by v-aangie** [comment]
@hamenon-ms — This issue appears to be unaddressed. Can you let me know if a docs update is needed? Thanks.

**Comment by hamenon-ms** [comment]
No. The Create adaptive card button is an experimental feature that isn't fully rolled out yet. The expected behavior is to paste that content into the message field for the action. 
![image](https://user-images.githubusercontent.com/39499084/117848289-0400b500-b238-11eb-8d39-19d8f99387db.png)

**Comment by v-aangie** [comment]
@MSFTMan - Verified by testing against doc that no changes related to the issue are needed. Updated label names to match UX and resized a screenshot.
